src/python/foglamp/data_purge/purge.py

```diff
@@ -176,115 +128,74 @@ def execute_command(stmt: str):
-        failed_removal_query = sqlalchemy.select([sqlalchemy.func.count()]).select_from(
-            table_name).where(table_name.c.ts <= age_timestamp).where(
-            table_name.c.ts <= start_time)
-        failed_removal_count = execute_command(failed_removal_query).fetchall()
-        failed_removal_count = int(failed_removal_count[0][0])
-        total_rows_removed = total_count_before - failed_removal_count
-
-    # Total number of rows that remain under start_time
-    total_count = sqlalchemy.select([sqlalchemy.func.count()]).select_from(table_name).where(
-        table_name.c.ts < start_time)
-    total_count_after = execute_command(total_count).fetchall()
-    total_count_after = int(total_count_after[0][0])
-
-    # Time  purge process finished
-    end_time = datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d %H:%M:%S')
-
-    """Levels:
-    -> 0 - No issues  (success) 
-    -> 1 - Rows failed to get removed (failure) 
-    -> 2 - (warnings)
-    --> less than 10% of original data remain in database  
-    --> unsent data was purged 
-    """
-    level = 0
-    unsent_count = execute_command(number_sent_rows_query).fetchall()
-    if failed_removal_count > 0:
-        level = 1
-    elif (int(unsent_count[0][0]) < unsent_rows_before) or (total_count_after <= int(total_count_before * 0.1)):
-        level = 2
-
+def purge(config, table_name):
     """Column information
     start_time - time that purge process began
     end_time - time that purge process ended
+    -------------------------------------------
     total_rows_removed - number of rows removed 
-    total_unsent_rows - total number of unsent rows under start_time 
-    total_failed_to_remove - total number of rows that failed to remove (this is the confirmation whether  purge 
-                                                                            succeeded or not.) 
+        - DELETE stmt returns value
+    ------------------------------------------
+    unsent_rows_removed - number of rows that weren't sent to historian, but were removed
+    failed_removal - number of rows that failed to remove
+        - SELECT stmt of DELETE
+    rows_remaining - total number of rows remain at screenshot 
+        - total_row - total_rows_removed
     """
-    unsent_rows_removed = unsent_rows_before-int(unsent_count[0][0]) 
-    if unsent_rows_removed < 0: 
-        unsent_rows_removed = 0
-    purge_set = {'start_time': start_time, 'end_time': end_time, 'rows_removed': total_rows_removed,
-                 'rows_remaining': total_count_after, 'unsent_rows_removed': unsent_rows_removed,
-                 'total_failed_to_remove': failed_removal_count}
+    start_time = time.strftime('%Y-%m-%d %H:%M:%S.%s', time.localtime(time.time()))
+
+    unsent_rows_removed = 0
+    total_rows_removed = 0
+    last_id = sqlalchemy.select([sqlalchemy.func.max(_STREAMS_TABLE.c.last_object)]).select_from(_STREAMS_TABLE)
```

Comment:
All the review comments say to use the MIN() of the objects ids in the streams table, so why is the sqlalchemy.func.max()?